statsmodels/tsa/arima_model.py

```diff
@@ -812,7 +827,7 @@ def loglike_css(self, params, set_sigma2=True):
         llf = -nobs/2.*(log(2*pi) + log(sigma2)) - ssr/(2*sigma2)
         return llf
 
-    def fit(self, start_params=None, trend='c', method="css-mle",
+    def fit(self, start_params=None, longar_maxlag=None, trend='c', method="css-mle",
             transparams=True, solver='lbfgs', maxiter=50, full_output=1,
             disp=5, callback=None, **kwargs):
```

Comment:
I  guess the new argument should go at the end (before `**kwargs`). Conceptually it's like an optimizer kwarg.
Also in the front it is too distracting and the user shouldn't have to worry about it under "normal" circumstances.